cmd/tapcli/universe.go

```diff
@@ -698,6 +715,218 @@ func universeFederationDel(ctx *cli.Context) error {
 	return nil
 }
 
+var universeFederationConfigCommand = cli.Command{
+	Name:      "config",
+	ShortName: "c",
+	Usage:     "Change the sync behavior of the local Universe",
+	Description: `
+	Manage the sync behavior of the local Universe. These settings are
+	defined by the proof type (issuance or transfer), the sync behavior
+	(insert from remote Universe or export to remote Universe), and the
+	scope (all assets or specific assets).
+        `,
+	Subcommands: []cli.Command{
+		universeFederationGlobalConfig,
+		universeFederationLocalConfig,
+	},
+}
+
+var (
+	proofInsertName        = "insert"
+	proofExportName        = "export"
+	universeSyncConfigArgs = []cli.Flag{
+		cli.StringFlag{
+			Name:  proofTypeName,
+			Usage: "the type of proof",
+		},
+		cli.StringFlag{
+			Name: proofInsertName,
+			Usage: "if true, remote Universes can push proofs to" +
+				"the local Universe",
+		},
+		cli.StringFlag{
```

Comment:
Why can't this be a `bool`? Or is that we want to actually allow it to not bet set (so you don't have to always set both)?

Reply:
Also that, I think it would make the most sense to be able to set either field or both with one command, vs one command per field.

Reply:
Also for `ctx.BoolFlag`, actually trying to resolve the flag returns `false` if parsing fails, instead of a separate error.